### PR TITLE
Add M2M template

### PR DESCRIPTION
**Pull request by amanda-ariyaratne**
### Purpose

This PR adds the M2M application template.

Screenshot 2023-10-18 at 18 26 38

Screenshot 2023-10-18 at 18 30 12

Screenshot 2023-10-18 at 18 29 20

Screenshot 2023-10-18 at 18 29 32

### Related Issues
- https://github.com/wso2/product-is/issues/17006

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/edit-application.tsx [hunk 1]
```diff
@@ -236,6 +238,19 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
         }
     },[ template ]);
 
+    /**
+     * Check whether the application is an M2M Application
+     */
+    useEffect(() => {
+        if (!template?.id) {
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/edit-application.tsx:245]:
```suggestion
        if (template) {    
```

File: apps/console/src/features/applications/components/edit-application.tsx [hunk 2]
```diff
@@ -236,6 +238,19 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
...
 
+    /**
+     * Check whether the application is an M2M Application
+     */
+    useEffect(() => {
+        if (!template?.id) {    
+            return;
+        }
+
+        if (template?.id == ApplicationTemplateIdTypes.M2M_APPLICATION) {
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/edit-application.tsx:249]:
```suggestion
        if (template.id == ApplicationTemplateIdTypes.M2M_APPLICATION) {
```

Comment (Achintha444) [reply apps/console/src/features/applications/components/edit-application.tsx:249]: Because we are capturing the `undefined` template in the line 245.

File: apps/console/src/features/applications/components/edit-application.tsx [hunk 3]
```diff
@@ -889,7 +908,7 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
             if (isFeatureEnabled(featureConfig?.applications,
                 ApplicationManagementConstants.FEATURE_DICTIONARY.get("APPLICATION_SHARED_ACCESS"))
                  && application?.templateId != ApplicationManagementConstants.CUSTOM_APPLICATION_PASSIVE_STS
-                    && !isFragmentApp
+                    && !isFragmentApp && !isM2MApplication
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/edit-application.tsx:911]:
```suggestion
                    && !isFragmentApp 
                    && !isM2MApplication
```

File: apps/console/src/features/applications/components/forms/general-details-form.tsx [hunk 4]
```diff
@@ -438,7 +446,8 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
                 /> ) : null
             }
 
-            { !featureConfig?.applications?.disabledFeatures?.includes("saasAppConfig") && (
+            { !featureConfig?.applications?.disabledFeatures?.includes("saasAppConfig") &&
+                !isM2MApplication && (
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/forms/general-details-form.tsx:450]:
```suggestion
            { !featureConfig?.applications?.disabledFeatures?.includes("saasAppConfig")
            && !isM2MApplication 
            && (
```

File: apps/console/src/features/applications/components/forms/general-details-form.tsx [hunk 5]
```diff
@@ -457,33 +466,37 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
...
+                        readOnly={
+                            !hasRequiredScope || (
+                                readOnly
+                                && applicationConfig.generalSettings.getFieldReadOnlyStatus(
+                                    application, "ACCESS_URL"
+                                )
+                            )
+                        }
+                        maxLength={ 200 }
+                        minLength={ 3 }
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/forms/general-details-form.tsx:493]: Keep min and max length in the constant file

File: apps/console/src/features/applications/components/forms/general-details-form.tsx [hunk 6]
```diff
@@ -457,33 +466,37 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
...
+                        data-testid={ `${ testId }-application-access-url-input` }
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/forms/general-details-form.tsx:494]: use `data-componentid` instead.

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 7]
```diff
@@ -414,6 +417,22 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         setIsLoading(false);
     }, [ template ]);
 
+    /**
+     * Check whether the application is an M2M Application
+     */
+    useEffect(() => {
+        if (!template?.id || !M2MApplicationTemplate?.id) {
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:424]:
```suggestion
        if (!template || !M2MApplicationTemplate) {
```

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 8]
```diff
@@ -414,6 +417,22 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
+            setIsLoading(false);
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:425]: shouldn't this be `setLoading(true)` ?

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 9]
```diff
@@ -414,6 +417,22 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
+
+            return;
+        }
+
+        if (template?.id == M2MApplicationTemplate?.id) {
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:430]:
```suggestion
        if (template.id == M2MApplicationTemplate.id) {
```

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 10]
```diff
@@ -1900,6 +1943,91 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
+                                    </Trans>
+                                ) }
+                                readOnly={ readOnly }
+                                addURLTooltip={ t("common:addURL") }
+                                duplicateURLErrorMessage={ t("common:duplicateURLError") }
+                                getSubmit={ (submitFunction: (callback: (url?: string) => void) => void) => {
+                                    submitUrl = submitFunction;
+                                } }
+                                showPredictions={ false }
+                                skipInternalValidation={ true }
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:2009]:
```suggestion
                                skipInternalValidation
```

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 11]
```diff
@@ -2006,97 +2134,104 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:2208]:
```suggestion
                                skipInternalValidation
```

**Comment by dasuni-30** [comment]
Let's add the changeset file as well.

File: apps/console/src/features/applications/components/forms/general-details-form.tsx [hunk 12]
```diff
@@ -259,6 +260,13 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
         isLoading: isMyAccountStatusLoading
     } = useMyAccountStatus(!isSubOrg);
 
+    /**
+     * Checks whether this is an M2M application.
+     */
+    useEffect(() => {
+        setM2MApplication(application.templateId === ApplicationManagementConstants.M2M_APP_TEMPLATE_ID);
```
Comment (pavinduLakshan) [inline apps/console/src/features/applications/components/forms/general-details-form.tsx:267]:
```suggestion
        setM2MApplication(application?.templateId === ApplicationManagementConstants.M2M_APP_TEMPLATE_ID);
```

File: apps/console/src/features/applications/components/forms/general-details-form.tsx [hunk 13]
```diff
@@ -457,33 +466,37 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
...
-                maxLength={ 200 }
-                minLength={ 3 }
-                data-testid={ `${ testId }-application-access-url-input` }
-                hint={ t("console:develop.features.applications.forms.generalDetails.fields.accessUrl.hint") }
-                width={ 16 }
-            />
+            { 
+                !isM2MApplication && (
+                    <Field.Input
+                        ariaLabel="Application access URL"
```
Comment (pavinduLakshan) [inline apps/console/src/features/applications/components/forms/general-details-form.tsx:472]: Shall we use the i18n label value for the ariaLabel too?

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 14]
```diff
@@ -1678,49 +1713,53 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
+                                default={
+                                    initialValues?.accessToken?.bindingType
+                                        ? initialValues.accessToken.bindingType
+                                        : metadata?.accessTokenBindingType?.defaultValue
+                                        ?? SupportedAccessTokenBindingTypes.NONE
+                                }
+                                type="radio"
+                                children={ getAllowedListForAccessToken(metadata.accessTokenBindingType, true) }
+                                readOnly={ readOnly }
+                                data-testid={ `${ testId }-access-token-type-radio-group` }
```
Comment (pavinduLakshan) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:1736]: Let's use `data-componentid` instead

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 15]
```diff
@@ -1798,60 +1837,64 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
+                                        : metadata.defaultUserAccessTokenExpiryTime
+                                }
+                                placeholder={
+                                    t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                        ".accessToken.fields.expiry.placeholder")
+                                }
+                                type="number"
+                                readOnly={ readOnly }
+                                min={ 1 }
+                                data-testid={ `${ testId }-access-token-expiry-time-input` }
```
Comment (pavinduLakshan) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:1879]: Let's use component id instead

File: apps/console/src/features/applications/components/settings/access-configuration.tsx [hunk 16]
```diff
@@ -776,6 +781,31 @@ export const AccessConfiguration: FunctionComponent<AccessConfigurationPropsInte
...
+                            icon={ getInboundProtocolLogos()[ SupportedAuthProtocolTypes.OAUTH2 ] }
+                            size="x30"
+                            verticalAlign="middle"
+                        />
+                        <Header.Content
+                            className={ "mt-1" }
+                        >
+                            <strong> {
+                                ApplicationManagementUtils.resolveProtocolDisplayName(SupportedAuthProtocolTypes.OAUTH2)
+                            } </strong>
```
Comment (pavinduLakshan) [inline apps/console/src/features/applications/components/settings/access-configuration.tsx:800]: WDYT?

```suggestion
                            <strong>
                                {  ApplicationManagementUtils.resolveProtocolDisplayName(SupportedAuthProtocolTypes.OAUTH2) }
                                </strong>
```

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx [hunk 17]
```diff
@@ -355,6 +356,18 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
             }
         }
 
+        // If the selected template is M2M
```
Comment (pavinduLakshan) [inline apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:359]: Shall we remove this comment as the code is self-explanatory?

File: apps/console/src/features/applications/components/edit-application.tsx (hunk 2 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/edit-application.tsx:249]: Agreed. Let's keep the optional chaining operator and remove the first if check.

File: apps/console/src/features/applications/components/edit-application.tsx (hunk 1 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/edit-application.tsx]: Removed this conditional check in 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b because the optional chaining operator was used.

File: apps/console/src/features/applications/components/edit-application.tsx (hunk 2 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/edit-application.tsx:246]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/edit-application.tsx (hunk 3 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/edit-application.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/forms/general-details-form.tsx (hunk 4 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/general-details-form.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/forms/general-details-form.tsx (hunk 5 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/general-details-form.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 7 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 8 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: Yes. 
Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 9 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 10 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 11 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/forms/general-details-form.tsx [hunk 18]
```diff
@@ -390,13 +398,13 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
                     hidden={ hiddenFields?.includes("imageUrl") }
                 />
             }
-            { (isMyAccountEnabled || isSubOrg) ? (
+            { (!isM2MApplication && (isMyAccountEnabled || isSubOrg)) ? (
```
[the remark this answers is not in the document]
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/general-details-form.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/forms/general-details-form.tsx (hunk 12 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/general-details-form.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/forms/general-details-form.tsx (hunk 13 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/general-details-form.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/settings/access-configuration.tsx (hunk 16 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/settings/access-configuration.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx (hunk 17 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx]: Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b

File: .changeset/hip-bees-pull.md [hunk 19]
```diff
@@ -0,0 +1,7 @@
+---
+"@wso2is/theme": patch
+"@wso2is/console": patch
+"@wso2is/i18n": patch
```
Comment (Achintha444) [inline .changeset/hip-bees-pull.md:4]: "@wso2is/myaccount": patch
"@wso2is/identity-apps-core": patch

add these two as well.

Comment (amanda-ariyaratne) [reply .changeset/hip-bees-pull.md:4]: Fixed with 53f7c98db34d722f89d371c8155ab48d1752b04a

**Review by Achintha444 — APPROVED** [review]

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 20]
```diff
@@ -785,6 +804,8 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                  */
                 const grant: CheckboxChildWithIndex = {
                     label: modifyGrantTypeLabels(name, displayName),
+                    readOnly: (isM2MApplication && name === ApplicationManagementConstants.CLIENT_CREDENTIALS_GRANT) 
+                        ? true : false,
```
Comment (JayaShakthi97) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:808]: can be simplified
```
isM2MApplication && name === ApplicationManagementConstants.CLIENT_CREDENTIALS_GRANT
```

File: apps/console/src/public/deployment.config.json [hunk 21]
```diff
@@ -354,7 +354,7 @@
                 }
             },
             "applications": {
-                "disabledFeatures": [],
+                "disabledFeatures": [""],
```
Comment (JayaShakthi97) [inline apps/console/src/public/deployment.config.json:357]: Is this change intended?

File: apps/console/src/features/applications/components/edit-application.tsx [hunk 22]
```diff
@@ -236,6 +238,16 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
         }
     },[ template ]);
 
+    /**
+     * Check whether the application is an M2M Application
+     */
+    useEffect(() => {
+
+        if (template?.id == ApplicationTemplateIdTypes.M2M_APPLICATION) {
```
Comment (ThaminduDilshan) [inline apps/console/src/features/applications/components/edit-application.tsx:246]:
```suggestion
        if (template?.id === ApplicationTemplateIdTypes.M2M_APPLICATION) {
```

File: apps/console/src/features/applications/components/edit-application.tsx [hunk 23]
```diff
@@ -236,6 +238,16 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
         }
     },[ template ]);
 
+    /**
+     * Check whether the application is an M2M Application
```
Comment (ThaminduDilshan) [inline apps/console/src/features/applications/components/edit-application.tsx:242]:
```suggestion
     * Check whether the application is an M2M Application.
```

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 24]
```diff
@@ -414,6 +417,22 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
+     * Check whether the application is an M2M Application
+     */
+    useEffect(() => {
+        if (!template || !M2MApplicationTemplate) {
+            setIsLoading(true);
+
+            return;
+        }
+
+        if (template.id == M2MApplicationTemplate.id) {
```
Comment (ThaminduDilshan) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:430]:
```suggestion
        if (template.id === M2MApplicationTemplate.id) {
```

File: apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx [hunk 25]
```diff
@@ -170,6 +172,17 @@ export const ApplicationCertificateWrapper: FunctionComponent<ApplicationWrapper
         }
     }, [ selectedCertType ]);
 
+    /**
+     * Set Is M2M Application
+     */
+    useEffect(()=> {
+
+        if (application?.templateId === ApplicationTemplateIdTypes.M2M_APPLICATION){
```
Comment (ThaminduDilshan) [inline apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx:180]:
```suggestion
        if (application?.templateId === ApplicationTemplateIdTypes.M2M_APPLICATION) {
```

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx [hunk 26]
```diff
@@ -355,6 +356,17 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
             }
         }
 
+        if (selectedTemplate.id === ApplicationTemplateIdTypes.M2M_APPLICATION) {
+            // M2M Apps are created with `client_credentials` grant by default.
+            application.inboundProtocolConfiguration = {
+                oidc: {
+                    grantTypes: [
+                        "client_credentials"
```
Comment (ThaminduDilshan) [inline apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:364]: Is it possible to use an already define constant here?

File: apps/console/src/features/applications/data/application-templates/templates/m2m-application/m2m-application.json [hunk 27]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (ThaminduDilshan) [inline apps/console/src/features/applications/data/application-templates/templates/m2m-application/m2m-application.json:1]: Do we need to add this to https://github.com/wso2/carbon-identity-framework/tree/master/features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/applications as well?

File: apps/console/src/public/deployment.config.json (hunk 21 above)
Comment (amanda-ariyaratne) [reply apps/console/src/public/deployment.config.json:357]: No, this has been removed in subsequent changes.

File: apps/console/src/features/applications/data/application-templates/templates/m2m-application/m2m-application.json (hunk 27 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/data/application-templates/templates/m2m-application/m2m-application.json:1]: I did not find a usage of adding these resources here.

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 14 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:1738]: data-testid was used to preserve the compatibility with tests

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 15 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:1881]: data-testid was used to preserve the compatibility with tests

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 20 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:808]: Fixed with https://github.com/wso2/identity-apps/pull/4605

File: apps/console/src/features/applications/components/edit-application.tsx (hunk 22 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/edit-application.tsx:246]: Fixed with https://github.com/wso2/identity-apps/pull/4605

File: apps/console/src/features/applications/components/edit-application.tsx (hunk 23 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/edit-application.tsx:242]: Fixed with https://github.com/wso2/identity-apps/pull/4605

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx (hunk 26 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:364]: Fixed with https://github.com/wso2/identity-apps/pull/4605

File: apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx (hunk 25 above)
Comment (amanda-ariyaratne) [reply apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx:180]: Fixed with https://github.com/wso2/identity-apps/pull/4605
